docs/reference/nuctl/nuctl.md

```diff
@@ -72,3 +72,50 @@ See [exposing a function](/docs/tasks/deploying-functions.md#exposing-a-function
 
 For your convenience, when deploying a function using `nuctl`, exposing it via a `NodePort` can be easily done by using the
 CLI arg `--http-trigger-service-type=nodePort`.
+
+
+### Redeploying functions
+
+Currently `redeploy` command is the only command which uses dashboard API. Namely, it uses `Patch` request.  
+Main difference of `redeploy` command from `deploy` command is that during redeployment we do not build image, which 
+means that image should be presented in registry. 
```

Comment:
Doesnt really matter how, but what so I would focus on
"Redeploy is the way to import function without building them. This is useful when function images are prebuilt and you change your platform (e.g. Kubernetes) environment. Backup and restore purposes, and so on.
In case a full deployment is needed, along with rebuilding the function images, use `nuctl deploy` option.